pkg/capr/planner/planner.go

```diff
@@ -868,6 +868,11 @@ func (p *Planner) reconcile(controlPlane *rkev1.RKEControlPlane, tokensSecret pl
 		return err
 	}
 
+	mgmCluster, err := p.managementClusters.Get(controlPlane.Spec.ManagementClusterName)
```

Comment:
don't leak the management cluster retrieval into the planner -- this should be put into an info function.